plugins/coverimage.koplugin/main.lua

```diff
@@ -18,8 +21,34 @@ local util = require("util")
 local _ = require("gettext")
 local T = require("ffi/util").template
 
+local PathChooser = require("ui/widget/pathchooser")
+local Screen = require("device").screen
+local InputDialog = require("ui/widget/inputdialog")
+local realpath = ffiutil.realpath
+
+-- todo: please check the default paths directly on the depending device
+-- todo: Wouldn't we like an android.deviceIdentifier() method, so we can use better default paths?
+local function getDefaultCoverPath()
+    if Device.isEmulator() then
+        return DataStorage:getDataDir() .. "/"
+    elseif Device.isAndroid() then
+        if android.prop.product == "ntx_6sl" then -- Tolino HD4 and other
+            return android.getExternalStoragePath() .. "/suspend_others.jpg"
+        else
+            return "cover.jpg"
+        end
```

Comment:
Yeah, for now, I'd probably move that to a method in Android's Device implementation, maybe?

Reply:
Speaking of: We traditionally use these as methods, not functions (e.g., Device*:*isWhatever()`)

Reply:
Good idea. We could make a device specific method `device.defaultCover()` which delivers a sane cover path (if known)